composer.json

```diff
@@ -1,41 +1,76 @@
 {
 	"name": "rtcamp/oneupdate",
-	"version": "1.0.0",
-	"description": "OneUpdate - Enterprise WordPress Plugin Manager Automate plugin updates across multiple WordPress sites with CI/CD integration. Creates pull requests for seamless development-to-production workflows.",
 	"type": "wordpress-plugin",
-	"autoload": {
-		"classmap": [
-			"inc/"
-		],
-		"files": [
-			"inc/helpers/custom-functions.php"
-		]
-	},
 	"authors": [
 		{
-			"name": "Utsav Patel",
-			"email": "up1512001@gmail.com"
+			"name": "rtCamp",
+			"email": "marketing@rtcamp.com",
+			"homepage": "https://rtcamp.com"
 		}
 	],
-	"require-dev": {
-		"wp-coding-standards/wpcs": "^2.3",
-		"phpcompatibility/phpcompatibility-wp": "^2.1",
-		"automattic/vipwpcs": "^2.3",
-		"phpunit/phpunit": "^9.6",
-		"wp-phpunit/wp-phpunit": "^6.2",
-		"yoast/phpunit-polyfills": "^1.0"
-	},
 	"config": {
 		"allow-plugins": {
-			"dealerdirect/phpcodesniffer-composer-installer": true
+			"dealerdirect/phpcodesniffer-composer-installer": true,
+			"phpstan/extension-installer": true,
+			"composer/installers": true
+		},
+		"platform": {
+			"php": "8.1"
```

Comment:
This is intentionally PHP 8.1 and not 8.0? If so we should update the docs + plugin headers + phpcs/phpstan configs. If not, we should drop this + the `require` below.

Reply:
Okay, make sure to bump readme.txt and remove it from the (commented out) phpunit matrix in test.yml